### PR TITLE
Add japanese language for LB4

### PR DESCRIPTION
**Pull request by altyjp**
Hello

I created a Japanese version of the document.
Would it be possible to merge this?

Thank you.

### CODE REVIEW

**Comment by dhmlau** [comment]
@nabdelgadir, could you please help land this PR? Thanks.

File: _data/topnav.yml [hunk 1]
```diff
@@ -64,3 +64,5 @@ topnav_dropdowns:
           url: /doc/zh/lb2/
         - title: "Français (v4)"
           url: /doc/fr/lb4/
+        - title: "日本語 (v4)"
+          url: /doc/ja/lb4/
```
Comment (nabdelgadir) [inline _data/topnav.yml]: Please add a new line to the end of the file.

File: _data/sidebars/ja_lb4_sidebar.yml [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+# title: string
+# url: /doc/ja/lb4/string
+# output: [ enum{web, pdf} ] (expressed as comma separated string in YAML)
+# children?: [ menuitem ]
+
+## top level property=================
+# entries: [ menuitem ]
+
+title: LoopBack 4
+url: /doc/ja/lb4/index.html
```
Comment (nabdelgadir) [inline _data/sidebars/ja_lb4_sidebar.yml:11]: Can you add `translated: true` here?

**Review by nabdelgadir — COMMENTED** [review]
LGTM mostly, but going on a page that isn't translated e.g. `/doc/ja/lb4/Getting-started.html` gives `Page not found` instead of the `Not yet translated` page. I think you might have to add `translated: false` under the sidebar entires.

**Comment by ryo-sato-oct22** [comment]
@nabdelgadir 
Thanks for reviewing.

I fixed those problem.
(but I mistook a merge for a rebase. so I force-pushed for undo the merge commits. )

Please confirm the PR.

**Review by nabdelgadir — APPROVED** [review]
LGTM! Can you please squash your commits to one commit and then I'll land it?

**Comment by ryo-sato-oct22** [comment]
@nabdelgadir 
OK!
I've finished!

**Comment by nabdelgadir** [comment]
@altyjp Landed, thanks for the contribution 🎉

**Pull request merged**